### DISCUSSION TITLE
DVCC - is there a way to only activate the current limit above a certain SOC

### DISCUSSION THREAD

**Discussion by merlep** [Louisvdw/dbus-serialbattery#201] [Q&A] [answered]
IS there a way to only apply the DVCC current limit above/ below certain SOC values ? This is the commercial batterys work.

**Comment by Louisvdw** [comment, marked as answer]
You always have a DVCC current limit. What that limit is, is up to you and the CCCM feature of the driver will do the limits above/below SOC values.
https://github.com/Louisvdw/dbus-serialbattery/wiki/Features#charge-current-control-management

**Comment by merlep** [comment]
IS there a way to disable CCCM, I have developed my own settings over time to manage my Lithium battery to charge to about 90%soc only and once a month I charge to "full " charge ( for me its 3.425V @ < 1 A ) . I do currently use DVCC ( with no battery BMS interface ) as it allows the use on the BMV702 as the reference voltage for my 3 battery system.  If I enable CCCM it will always certainly always charge my battery to100 % which  I do not want,

**Reply by Louisvdw** [reply]
You can disable it in utils.py
> CCCM_ENABLE = False
